modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -3,7 +3,7 @@
 /// <reference path="{{apiRelativeToRoot}}custom.d.ts" />
 {{>licenseInfo}}
 
-import * as url from 'url';
+import * as $url from 'url';
```

Comment:
instead of renaming the parameter, I think it is better to add `url` to the reserved keywords. See e.g. https://github.com/OpenAPITools/openapi-generator/blob/1b1c13db0fa923a059b234da1d92dd98c72d9aaa/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java#L262-L263

Reply:
@macjohnny yup, that's one way to do it.

My little suggestion is to prefix it (like what's done in the PR) in this case as "url" may be a popular choice for naming variables/classes.

Reserved keywords are usually language specified such as return in PHP that can must be prefixed/suffixed.

Reply:
but someone might also use `url$` as a parameter name. I think it is a good idea to add all global symbols to the reserved keywords to avoid any naming conflicts.

Reply:
I think the dollar sign and other symbols will be removed in toVarName or toParamName as part of the sanitization.

what about using the "localVar" prefix approach?

localVar may not sound right in this case so what about "globalImportUrl"?

Reply:
"globalImportUrl" seems fine.

Reply:
@vellengs please go ahead with this approach to see if it works for you (I think it should).

Reply:
@wing328  can you build the shell 
./bin/typescript-axios-petstore-all.sh.
I have promblem with Marven.

Reply:
Can you use maven wrapper instead?
```
./mvnw clean install
```

Reply:
@naelrashdeen I changed two part of  'api.mustache' and 'apiInner.mustache'
why the build samples file still have  
```
import * as url from 'url';
```